hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azure/SyncableDataOutputStream.java

```diff
@@ -24,6 +24,8 @@
 
 import org.apache.hadoop.fs.StreamCapabilities;
 import org.apache.hadoop.fs.Syncable;
+import org.slf4j.Logger;
```

Comment:
put in an import block just above the org.apache one. I know a lot of classes don't do that -but that is because the move from o.a.commons.logging to SLF4J was done with 'sed'

Reply:
Done. 
org.apache.hadoop.classification.InterfaceAudience was wrongly placed.  Corrected it now.

Reply:
sorry, i meant "org.slf4j to go into a block above the org.apache one", below the java. one...its where we like  keep the non-apache, non-static imports